jekyll/_cci2/creating-orbs.md

```diff
@@ -438,9 +440,9 @@ You should see a response similar to the following:
 Config file at .circleci/config.yml is valid
 ```
 
-### Set the Version Property to 2.1
+### Bump Version Property to Orb-Compatible 2.1
```

Comment:
This needs to be prefixed with `Step 5 - ` for consistency

Reply:
Re-numbered all steps based on new workflow diagram